lib/cog/commands/help/command_formatter.ex

```diff
@@ -1,74 +1,34 @@
 defmodule Cog.Commands.Help.CommandFormatter do
```

Comment:
Also, I notice there are a bunch of `render_X` functions here, but I was expecting to see maybe one public function `render` that takes a command and coordinates the whole conversion, with other functions being private.